hack/actions/build-and-push-release-images.sh

```diff
@@ -0,0 +1,56 @@
+#! /usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# This scripts sets TAG_LATEST based on whether the tag currently being
+# built is the highest semantic version tag that's not a pre-release
+# (alpha, beta, rc), and calls 'make push'.
+#
+# This script is intended to be run only for tag builds and will no-op
+# if GITHUB_REF is not in the format "refs/tags/<tag-name>".
```

Comment:
Not sure how stable the link is, but it would be handy to link to the docs for GITHUB_REF

https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables